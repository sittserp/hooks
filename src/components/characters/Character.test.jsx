import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Character from './Character';
import { MemoryRouter } from 'react-router-dom';

describe('Character component', () => {
  afterEach(() => cleanup());
  it('renders Character', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Character />
      </MemoryRouter>);
    expect(asFragment()).toMatchSnapshot();
  });
});
