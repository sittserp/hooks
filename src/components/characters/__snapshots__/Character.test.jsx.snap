// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Character component renders Character 1`] = `
<DocumentFragment>
  <a
    href="/details/undefined"
  >
    <figure>
      <img />
      <figcaption />
    </figure>
  </a>
</DocumentFragment>
`;
